ygnmi/bench_test.go

```diff
@@ -271,3 +271,39 @@ func BenchmarkWatchAll(b *testing.B) {
 		return ygnmi.Continue
 	}).Await()
 }
+
+func BenchmarkLookupListParent(b *testing.B) {
+	sc := &dynamicClient{
+		numResponses: b.N,
```

Comment:
This reads me to mean benchmarking the performance of each response **as if each response had the same processing time**, but I'm not sure if we can assume that.

And if we can't, then I think the right thing might be to add a flag for `numResponses`, and run ygnmi.Lookup b.N times instead:
```
The benchmark function must run the target code b.N times. During benchmark execution, b.N is adjusted until the benchmark function lasts long enough to be timed reliably.
```

On my machine running without `-benchtime=2000x` is taking a long time, likely because it's adjusting b.N